lib/my_john_deere_api/model/machine_engine_hour.rb

```diff
@@ -0,0 +1,16 @@
+module MyJohnDeereApi
+  class Model::MachineEngineHour < Model::Base
+    attr_reader :reading, :report_time
+
+    private
+
+    def map_attributes(record)
+      @reading = record['reading']
+      @report_time = record['reportTime']
+    end
+
+    def expected_record_type
+      'EngineHours'
```

Comment:
the record type is plural but the model name is singular, which is correct?

Reply:
Yes, the API is returning the type as "EngineHours"